EESSI-extend-easybuild.eb

```diff
@@ -35,6 +35,11 @@ description = """
  If both EESSI_USER_INSTALL and EESSI_PROJECT_INSTALL are defined, both sets of
  installations are exposed, but new installations are created as user
  installations.
+
+ Strict installation path checking is enforced by EESSI for EESSI and site
+ installations involving accelerators. In these cases, if you wish to create an
+ accelerator installation you must set the environement variable
+ EESSI_ACCELERATOR_INSTALL (and load/reload this module).
```

Comment:
This new environment variable has an impact on the build scripts, it needs to be set in the scenario where we expect to do an accelerator installation